test-integration/student/dashboard.js

```diff
@@ -62,7 +62,7 @@ describe('Student Dashboard', function(){
     expect(unstartedEvents).not.to.be.empty;
 
     // that's all folks
-    this.student.logout();
+    // this.student.logout();
 
```

Comment:
Should test be moved to the 'long tests' section instead of being `xit`. Shouldn't we still logout the user?